initial_data.json

```diff
@@ -0,0 +1 @@
+[{"model": "blocks.block", "pk": 1, "fields": {"hash": "bfc88dc7b13f2afe8f244428d77ea41764bf7459", "merkle_hash": "47079671b913640d440a669c2219d2c777d41f2a", "prev_block": null, "timestamp": "2022-08-17T06:05:10.927Z", "height": 0, "num_transactions": 1, "hash_target_zeros": 0, "nonce": "genesis_block"}}, {"model": "transactions.transaction", "pk": 1, "fields": {"hash": "06c2fbd819915dd284e07657d564b459ddcb51bc", "block": 1, "tx_inputs_count": 0, "tx_outputs_count": 10, "mined": true}}, {"model": "transactions.transactionoutput", "pk": 1, "fields": {"hash": "ec0c1184e33bc40a9533c9701ed3499fcca826dc", "value": 100, "own_addr": "e17be4a47a4e9c1e7a430a9b1538f6b5c9e315d25ae7f48a39c525010675a2bcc86172ce07475ed8791ed8bf802f64c1", "gen_transaction_index": 0, "gen_transaction": 1, "blk_height": 0}}, {"model": "transactions.transactionoutput", "pk": 2, "fields": {"hash": "d1c58fe74cdd82c1922f18bbf068eada947496d7", "value": 100, "own_addr": "12569224227aec086a8dd83d96e3dbce063575edc5fe098b538884b51b1366466946a8ad6511104ca5342214b2e005b1", "gen_transaction_index": 1, "gen_transaction": 1, "blk_height": 0}}, {"model": "transactions.transactionoutput", "pk": 3, "fields": {"hash": "4f90f5fe21e1b06fd77833a2943afdbf28a5974e", "value": 100, "own_addr": "1ba2b749d96838d1c408050e6183acd87826a8108001f40e85148332be7fba1737abe05f7dc94b282ba487fb67d4addb", "gen_transaction_index": 2, "gen_transaction": 1, "blk_height": 0}}, {"model": "transactions.transactionoutput", "pk": 4, "fields": {"hash": "e704b129bb9cfde18f7893f4f08308d1d8b039e4", "value": 100, "own_addr": "737ef40a90e85ba0c59943c47be694634eadcf4e67eda2d01c79fc187ea2e52f71c6bb8ddcfd4fbdbfa60817c5489b6a", "gen_transaction_index": 3, "gen_transaction": 1, "blk_height": 0}}, {"model": "transactions.transactionoutput", "pk": 5, "fields": {"hash": "44a958c9be6667292d84b446aa845af6ec04e669", "value": 100, "own_addr": "5f9dfce9924918292fbc9848721f0d5851c8cebccf963d79a496dc3923dcaeb5d9d79e35cc616c9c25dd9deaaf61e6c5", "gen_transaction_index": 4, "gen_transaction": 1, "blk_height": 0}}]
```

Comment:
Can we add this in some database other than JSON? Having a PK in a public json file isn't good.

Reply:
this is our "genesis" block that is why it is public...everyone has to use it as their first block

Reply:
Suggestion: how about we add it to __init__() method? (NOT main)
https://www.section.io/engineering-education/how-to-create-a-blockchain-in-python/